selfdrive/car/toyota/interface.py

```diff
@@ -163,14 +163,14 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), has_relay=False,
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.17], [0.03]]
       ret.lateralTuning.pid.kf = 0.00006
 
-    elif candidate == CAR.RAV4_TSS2:
+    elif candidate in [CAR.RAV4_TSS2, CAR.RAV4H_TSS2]:
       stop_and_go = True
       ret.safetyParam = 73
       ret.wheelbase = 2.68986
       ret.steerRatio = 14.3
       tire_stiffness_factor = 0.7933
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.6], [0.1]]
-      ret.mass = 3370. * CV.LB_TO_KG + STD_CARGO_KG
+      ret.mass = 3585. * CV.LB_TO_KG + STD_CARGO_KG # mean between normal and hybrid
```

Comment:
Huh, why do the mean here? Can't it be an if?

Reply:
Good point. This was just copied from how other hybrids were done in the past. Fixed.

Reply:
Huh, yea, it's poorly done. This is good, or perhaps inline if if nothing else changes?